xfields/beam_elements/beambeam_src/beambeam3d.h

```diff
@@ -6,23 +6,34 @@
 #ifndef XFIELDS_BEAMBEAM3D_H
 #define XFIELDS_BEAMBEAM3D_H
 
+//#if !defined(dump_path)
+//    #define dump_path "/Users/pkicsiny/phd/cern/xsuite/outputs/n74c" 
+//#endif
```

Comment:
Remember to remove these commented lines

Reply:
removed